### DISCUSSION TITLE
adding new Third party(wechat) providers

### DISCUSSION THREAD

**Discussion by leohxj** [nhost/nhost#1221] [Ideas]
I am a developer from China,  WeChat is the most popular IM here, can we support it?
discuss with @elitan in Discord, he said that nhost is power by Grant, so we can add wechat easy.

here is some links resources:
- https://mp.weixin.qq.com/
- https://github.com/simov/grant

**Comment by 18601673727** [comment]
This could be a really nice improvement, because I'm thinking using nhost to build something serious, and without WeChat login that's just not possible here.
